odl_video/logging.py

```diff
@@ -0,0 +1,47 @@
+"""Configure structured logging for our application"""
+import logging
+from structlog_sentry import SentryJsonProcessor
+import structlog
+from odl_video import settings
+
+
+logging.basicConfig(level=getattr(logging, settings.LOG_LEVEL),
+                    format='%(message)s')
+
+structlog_processors_per_debug = {
+    True: [
+        structlog.stdlib.add_log_level,
+        structlog.stdlib.add_logger_name,
+        structlog.processors.StackInfoRenderer(),
+        structlog.dev.set_exc_info,
+        structlog.processors.format_exc_info,
+        structlog.processors.TimeStamper(fmt='iso'),
+        structlog.dev.ConsoleRenderer()
+    ],
+    False: [
+        structlog.stdlib.add_logger_name,
+        structlog.stdlib.add_log_level,
+        structlog.stdlib.filter_by_level,
+        structlog.processors.TimeStamper(),
+        SentryJsonProcessor(level=logging.ERROR,
+                            tag_keys=['environment', 'level', 'logger',
+                                      'runtime', 'server_name', 'video_id',
+                                      'video_hexkey', 's3_object_key',
+                                      'filename', 'youtubevideo_id',
+                                      'youtubevideo_video_id'],
+                            as_extra=False),
+        structlog.processors.StackInfoRenderer(),
+        structlog.processors.format_exc_info,
+        structlog.processors.JSONRenderer()
+    ]
+}
+
+structlog.configure(
+    processors=structlog_processors_per_debug[settings.DEBUG],
+    logger_factory=structlog.stdlib.LoggerFactory()
```

Comment:
if you add 
```python
wrapper_class=structlog.stdlib.BoundLogger,
```

then you won't have to replace occurrences of `log.exception('oops')` in master with `log.error('oops', exc_info=True)` in this branch.

Reply:
OK, changed in commit f9feae2